prov/gni/test/mr.c

```diff
@@ -140,7 +140,7 @@ TestSuite(memory_registration_bare, .init = mr_setup, .fini = mr_teardown);
 TestSuite(memory_registration_cache, .init = mr_setup, .fini = mr_teardown);
 
 TestSuite(perf_memory_registration, .init = mr_setup, .fini = mr_teardown,
-		.disabled = true);
+		.disabled = false);
```

Comment:
FYI I might end up disabling these for valgrind testing.  Just want to confirm (again) that these tests still do not test any functionality or code paths that the other tests cover.